resource/closecaption_german.txt

```diff
@@ -146,54 +146,54 @@
+"bastille.InjuredSelection3"		"<clr:205,127,50>Bastille: Die ganze Anlage geht den Bach runter!"
 "[english]bastille.InjuredSelection3"		"<clr:205,127,50>Bastille: This whole place is getting to hell!"
-"bastille.InjuredSelection4"		"<clr:205,127,50>Bastille: Was willst du jetzt?"
+"bastille.InjuredSelection4"		"<clr:205,127,50>Bastille: Was willst du denn jetzt schon wieder?"
 "[english]bastille.InjuredSelection4"		"<clr:205,127,50>Bastille: What do you want now?"
-"bastille.SeriousInjuryReply0"		"<clr:205,127,50>Bastille: Na gut, wenn du das sagst."
+"bastille.SeriousInjuryReply0"		"<clr:205,127,50>Bastille: Na gut, wenn du meinst."
 "[english]bastille.SeriousInjuryReply0"		"<clr:205,127,50>Bastille: All right, if you say so."
-"bastille.SeriousInjuryFollowUp0"		"<clr:205,127,50>Bastille: Ich will keine Stunts, ok?"
+"bastille.SeriousInjuryFollowUp0"		"<clr:205,127,50>Bastille: Keine Stunts mehr, ok?"
 "[english]bastille.SeriousInjuryFollowUp0"		"<clr:205,127,50>Bastille: I don't want any stunts, ok?"
-"bastille.SeriousInjuryFollowUp1"		"<clr:205,127,50>Bastille: Lass es ruhig angehen, das hat der Arzt angeordnet."
+"bastille.SeriousInjuryFollowUp1"		"<clr:205,127,50>Bastille: Lass es ruhig angehen, ärztliche Anordnung."
 "[english]bastille.SeriousInjuryFollowUp1"		"<clr:205,127,50>Bastille: Just take it easy, doctor's orders."
-"bastille.SeriousInjury0"		"<clr:205,127,50>Bastille: Sei vorsichtig, die Wunde könnte noch offen sein."
+"bastille.SeriousInjury0"		"<clr:205,127,50>Bastille: Sei vorsichtig, sonst öffnet sich die Wunde wieder."
 "[english]bastille.SeriousInjury0"		"<clr:205,127,50>Bastille: Be careful, the wound might still open."
-"bastille.SeriousInjury1"		"<clr:205,127,50>Bastille: Gute Nachrichten, du bist bei der nächsten Mission nicht dabei."
+"bastille.SeriousInjury1"		"<clr:205,127,50>Bastille: Gute Nachrichten, du bist von der nächsten Mission befreit."
 "[english]bastille.SeriousInjury1"		"<clr:205,127,50>Bastille: Good news, you're off the next mission."
-"bastille.StillBreathingReply0"		"<clr:205,127,50>Bastille: Du weißt, dass du dafür eine Beratung bekommen kannst, oder?"
+"bastille.StillBreathingReply0"		"<clr:205,127,50>Bastille: Du weißt, dass du dafür Therapie in Anspruch nehmen kannst, oder?"
 "[english]bastille.StillBreathingReply0"		"<clr:205,127,50>Bastille: You know you can get counseling for that, right?"
 "bastille.Use0"		"<clr:205,127,50>Bastille: Auf jeden Fall."
 "[english]bastille.Use0"		"<clr:205,127,50>Bastille: Absolutely."
 "bastille.Use1"		"<clr:205,127,50>Bastille: Aber natürlich."
 "[english]bastille.Use1"		"<clr:205,127,50>Bastille: But of course."
-"bastille.Use2"		"<clr:205,127,50>Bastille: Gute Idee."
+"bastille.Use2"		"<clr:205,127,50>Bastille: Gut mitgedacht."
 "[english]bastille.Use2"		"<clr:205,127,50>Bastille: Good thinking."
-"bastille.Use3"		"<clr:205,127,50>Bastille: Unverzüglich."
+"bastille.Use3"		"<clr:205,127,50>Bastille: Immediatement."
```

Comment:
Ich mag das der Franzose auch mal was in seiner Sprache beantworted. Gut mitgedacht. Aber wenn wir schon so genau sind gehen wir gleich in die Vollen und nehmen die Schreibung mit Akut (über dem e).
Immédiatement.